Gemfile

```diff
@@ -10,30 +10,19 @@ RUBY_ENGINE = 'ruby' unless defined? RUBY_ENGINE
 source 'https://rubygems.org' unless ENV['QUICK']
 gemspec
 
-gem 'rake'
+gem 'rake', '~> 10.0'
 gem 'rack-test', '>= 0.6.2'
 gem "minitest", "~> 5.0"
 
-# Allows stuff like `tilt=1.2.2 bundle install` or `tilt=master ...`.
```

Comment:
My only concern with removing this is that someone might be using it in production.